testutil-server/src/main/java/io/spine/testing/server/blackbox/BlackBoxBoundedContext.java

```diff
@@ -145,11 +152,19 @@
 
     private final MemoizingObserver<Ack> observer;
 
+    /**
+     * A guard that verifies that unsupported commands do not get posted to the
+     * {@code BlackBoxBoundedContext}.
+     */
+    private final UnsupportedGuard unsupportedGuard;
```

Comment:
Right now it looks like a guard, which isn't supported for some reason.

Let's rename the class to include `Command`. E.g. `UnsupportedCommandGuard`.